log_cache_test.go

```diff
@@ -137,8 +137,8 @@ func TestLogCacheWithBackendStoreError(t *testing.T) {
 
 	var out Log
 	for i := 1; i < 5; i++ {
-		if err := c.GetLog(uint64(i), &out); err != nil {
-			t.Fatalf("err: %v", err)
+		if e := c.GetLog(uint64(i), &out); e != nil {
+			t.Fatalf("err: %v", e)
```

Comment:
What's the reason for this change? Generally `err` is the correct variable name to use for errors. If it's because it's masking `err` from the parent scope, I think that's not really a concern in this test.